src/App.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Routes, Route, Navigate } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import Navbar from './components/Navbar';
+import Authentication from './components/Authentication';
+import SignIn from './components/auth/SignIn';
+import SignUp from './components/auth/SignUp';
+import Home from './components/home/Home';
+import DetailsPage from './components/details/detailsPage';
+import MyReservations from './components/reservations/reservations';
+import ReservationForm from './components/forms/ReservationForm';
+import ResortForm from './components/forms/resortForm';
+import ResortDelete from './components/forms/DeleteResort';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:

![image](https://github.com/Leeoasis/resort-booking-frontend-app/assets/57421823/fa0425ca-3b06-4653-8df0-a7488f211ae1)


That way, your code will meet the latest standards. 💪

Reply:
issue solved.